.drone.yml

```diff
@@ -10,7 +10,8 @@ pipeline:
   fmt_and_lint:
     image: python:3.6-alpine
     commands:
-      - pip install black==19.3b0 pylama
+      - pip install black==19.3b0 mypy=0.770 pylama
+      - mypy .
```

Comment:
I wonder if `mypy` was in some way related to the failure for the last build to run. If it did run, would it have passed? My gut says it would be better to hold off on introducing `mypy` - moving that into a separate PR. What are your thoughts?